zmon-controller-app/src/main/java/org/zalando/zmon/service/impl/ForceAlertEvaluation.java

```diff
@@ -0,0 +1,95 @@
+package org.zalando.zmon.service.impl;
+
+import java.util.UUID;
+
+import org.springframework.dao.DataAccessException;
+import org.springframework.data.redis.connection.RedisConnection;
+import org.springframework.data.redis.connection.StringRedisConnection;
+import org.springframework.data.redis.core.RedisCallback;
+import org.springframework.data.redis.core.RedisOperations;
+import org.springframework.util.Assert;
+import org.zalando.zmon.event.ZMonEventType;
+import org.zalando.zmon.redis.RedisPattern;
+import org.zalando.zmon.security.permission.DefaultZMonPermissionService;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+/**
+ * 
+ * @author jbellmann
+ *
+ */
+public class ForceAlertEvaluation {
+
+    private static final int INSTANTANEOUS_ALERT_EVALUATION_TIME = 300;
+
+    private final RedisOperations<String, String> redisOperations;
+
+    private final DefaultZMonPermissionService authorityService;
+
+    private final NoOpEventLog eventLog;
+
+    private final ObjectMapper objectMapper;
+
+    public ForceAlertEvaluation(RedisOperations<String, String> redisOperations,
+            DefaultZMonPermissionService authorityService) {
+
+        Assert.notNull(redisOperations, "'redisOperations' should not be null");
+        Assert.notNull(authorityService, "'authorityService' should not be null");
+
+        this.redisOperations = redisOperations;
+        this.authorityService = authorityService;
+
+        //
+        this.eventLog = new NoOpEventLog();
+        this.objectMapper = new ObjectMapper();
+    }
+
+    public void forceAlertEvaluation(final int alertDefinitionId) {
```

Comment:
Why do we need this? The open source controller triggers this via http to scheduler?